tests/NewCypress/cypress/pages/trusts/academiesInTrustPage.ts

```diff
@@ -0,0 +1,85 @@
+class AcademiesInTrustPage {
+
+    elements = {
+        DetailsPage: {
+            table: () => cy.get('.govuk-main-wrapper > .dfe-width-container > .govuk-grid-row > .govuk-grid-column-three-quarters'),
```

Comment:
This doesn't select the table, it selects all the content around it too. you could use `cy.get('table)` instead

Reply:
Now updated to use cy.get('table')